app/views/Home.scala.html

```diff
@@ -161,10 +148,10 @@
                     @link(controllers.agent.routes.RemoveClientDetailsSessionsController.show.url, "govuk-link--no-visited-state", "home.agent.changeClientLink")
                 }
 
-                @if(overDuePaymentsCount.getOrElse(0) > 0) {
+                @if(viewModel.nextPaymentsTileViewModel.overDuePaymentsCount.getOrElse(0) > 0) {
```

Comment:
we might need to change this in the future to something like:
`@if(viewModel.nextPaymentsTileViewModel.overDuePaymentsCount.nonEmpty) {`